packages/host/webpack.config.js

```diff
@@ -1,17 +1,22 @@
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+const webpack = require('webpack');
+const HotModuleReplacementPlugin = webpack.HotModuleReplacementPlugin;
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const ModuleFederationPlugin =
-  require('webpack').container.ModuleFederationPlugin;
+// const ModuleFederationPlugin = webpack.container.ModuleFederationPlugin;
```

Comment:
I commented this out.. it's likely we'll still need to use federation for plugins/later..